lib/library/movies_widgets_package.dart

```diff
@@ -0,0 +1,7 @@
+library movies_widgets_package;
+export '../widgets/movie_detail_image.dart';
+export '../widgets/movie_detail_play_button.dart';
+export '../widgets/movie_details_actions.dart';
+export '../widgets/movie_details_info.dart';
+export '../widgets/movie_details_overview.dart';
+export '../widgets/movie_details_title.dart';
```

Comment:
You missed a newline at end of file here

Reply:
Added